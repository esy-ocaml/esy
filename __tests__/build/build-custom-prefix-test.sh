doTest () {
  initFixture custom-prefix
  unset ESY__PREFIX
  run esy build
  expectStdout "custom-prefix" esy x custom-prefix
}
