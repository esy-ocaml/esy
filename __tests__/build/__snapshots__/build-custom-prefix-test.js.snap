// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`build custom-prefix 1`] = `
"RUNNING: esy config ls
esy:bin using <sandbox>/.esyrc
esy:bin prefix: <testRoot>/esy
RUNNING: esy build
esy:bin using <sandbox>/.esyrc
esy:bin prefix: <testRoot>/project/store
esy:bin needRebuildTarget <sandbox>/node_modules/.cache/_esy/build/bin/build
esy:bin trying to acquire lock
esy:bin:esyBuild execute
esy:bin trying to acquire lock
esy:bin using <sandbox>/.esyrc
esy:bin prefix: <testRoot>/project/store
esy:build-dependencies checking if dependencies are built
RUNNING: esy x custom-prefix
esy:bin using <sandbox>/.esyrc
esy:bin prefix: <testRoot>/project/store
esy:bin needRebuildTarget <sandbox>/node_modules/.cache/_esy/build/bin/build
esy:bin trying to acquire lock
esy:bin using <sandbox>/.esyrc
esy:bin prefix: <testRoot>/project/store
esy:build-dependencies checking if dependencies are built"
`;
